src/Utils/TypeInfo.php

```diff
@@ -40,37 +40,39 @@
 use GraphQL\Type\Introspection;
 use GraphQL\Type\Schema;
 use SplStack;
+use Symfony\Component\Console\Output\Output;
 use function array_map;
 use function array_merge;
 use function array_pop;
 use function count;
 use function is_array;
+use function is_null;
 use function sprintf;
 
 class TypeInfo
 {
     /** @var Schema */
     private $schema;
 
-    /** @var SplStack<OutputType> */
+    /** @var array<(OutputType&Type)|null> */
     private $typeStack;
```

Comment:
if it's not stack anymore, can be renamed. `$types`?

Reply:
Hmm, I dunno--I didn't change the run time type of this in any way, so it's still whatever it used to be. Do you still want me to rename it?